src/topp/FeatureFinderMultiplex.cpp

```diff
@@ -230,6 +231,8 @@ class TOPPFeatureFinderMultiplex :
       defaults.setMinInt("missed_cleavages", 0);
       defaults.setValue("knock_out", "false", "Is it likely that knock-outs are present? (Supported for doublex, triplex and quadruplex experiments only.)", ListUtils::create<String>("advanced"));
       defaults.setValidStrings("knock_out", ListUtils::create<String>("true,false"));
+      defaults.setValue("spectrum_type", "automatic", "Spectrum type of the MS level to be averaged", ListUtils::create<String>("advanced"));
```

Comment:
I think the description is not very clear. "Expected type of mass peaks in MS1 spectra" or something similar might be better.